### PR TITLE
Added a timing system!

### PR DESCRIPTION
**Pull request by RealNeGate**
* Added the Win32 implementation of timing.
* Added a time-delta used for real-time calculations.
* Added a customizable frame limiter.

### CODE REVIEW

**Review by netizen-a — COMMENTED** [review]
If your going to do Unicode you should do it properly...

**Comment by RealNeGate** [comment]
Low priority rn

File: src/Boxer/Platform/Win32/Win32Application.cpp [hunk 1]
```diff
@@ -110,14 +133,39 @@ namespace boxer {
...
 
 	void Application::Launch() {
 		Win32Handle* win32 = reinterpret_cast<Win32Handle*>(_Handle);
 
-		// Event handling
+		// Initialize Timer
+		LARGE_INTEGER frequency;
+		QueryPerformanceFrequency(&frequency);
+		m_Frequency = 1.0 / F64(frequency.QuadPart);
+		QueryPerformanceCounter(&m_Start);
```
Comment (travisvroman) [inline src/Boxer/Platform/Win32/Win32Application.cpp]: Not usually a fan of this being done inline. Perhaps clean this up by moving it to a private method and calling it instead. Launch should be as simple and lightweight as possible.

**Review by netizen-a — APPROVED** [review]
seems fine.

**Review by travisvroman — APPROVED** [review]

**Review by travisvroman — APPROVED** [review]

**Pull request merged**